docs/ref/contrib/sitemaps.txt

```diff
@@ -56,7 +56,7 @@ To activate sitemap generation on your Django site, add this line to your
 
     from django.contrib.sitemaps.views import sitemap
 
-    url(r'^sitemap\.xml$', sitemap, {'sitemaps': sitemaps},
+    url(r'^sitemap\.xml$', sitemap, {'sitemaps': sitemap},
```

Comment:
I this case I believe `sitemaps` should be a `dict` or an instance of `Sitemap` as described below, not the `sitemap` view function itself.